includes/html/graphs/port/upkts.inc.php

```diff
@@ -1,15 +1,17 @@
 <?php
 
+use LibreNMS\Config;
+
 $ds_in  = 'INUCASTPKTS';
 $ds_out = 'OUTUCASTPKTS';
 
-$colour_area_in  = 'AA66AA';
```

Comment:
Seems reasonable.
Did you know if you `Config::get('graph_colours.ports.upkts');`  You will get an array with keys of all the colors keyed by area_in, line_in, etc?

Reply:
Sorry not quite with you. Thought I had defined `.ports.upkts`

Reply:
Sorry I don't quite follow. Thought I had defined that path `.ports.upkts` and is not used anywhere else.

Reply:
The dot specifies a group.  You can get the parent group of all your colors to get them all at once in an array.

Also, looks like this needs a color picker option type.